haskell-src/Data/FixedByteString.hs

```diff
@@ -31,7 +31,7 @@ class FixedLength a where
 -- is determined by the type parameter @a@, which should be an
 -- instance of 'FixedLength', as @fixedLength (undefined :: a)@.
 newtype FixedByteString a = FixedByteString ByteArray
-  deriving(Data, Typeable)
+  deriving (Data, Typeable, Show)
```

Comment:
I think this deliberately did not have a Show, or rather the Show instance should not be used for this.